src/array-methods/11/task.js

```diff
@@ -0,0 +1,22 @@
+const vacationPackages = [
+    { destination: "Maldives", days: 10, cost: 2000 },
+    { destination: "Alaska", days: 5, cost: 1000 },
+    { destination: "Switzerland", days: 7, cost: 1500 }
+];
+
+let vacationPackagesWithAverageCost = vacationPackages.map(vacationPackage => ({
+    destination: vacationPackage.destination,
+    days: vacationPackage.days,
+    cost: vacationPackage.cost,
+    averageDailyCost: Math.round(vacationPackage.cost / vacationPackage.days),
+}));
```

Comment:
```suggestion
let vacationPackagesWithAverageCost = vacationPackages.map(vacationPackage => ({
    ...vacationPackage, // since you're not doing any changes on these elements
    averageDailyCost: Math.round(vacationPackage.cost / vacationPackage.days),
}));
```